db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
```

Comment:
What's the syntax for "saving the community into a variable"?  The "community_id: Community.first" syntax exemplified in step 7 at https://github.com/touchstone-cohousing/mess doesn't work:

~/mess$ rake db:fake_user_data
=> QueryTrace disabled; CTRL-\ to toggle
rake aborted!
ActiveRecord::RecordInvalid: Validation failed: Community can't be blank
/home/ken/mess/lib/tasks/fake_user_data.rake:11:in `block (2 levels) in <top (required)>'
Tasks: TOP => db:fake_user_data